src/Nethermind/Nethermind.Evm/Instruction.cs

```diff
@@ -172,9 +177,350 @@ public enum Instruction : byte
         INVALID = 0xfe,
         SELFDESTRUCT = 0xff,
     }
+    public struct OpcodeMetadata(long gasCost, byte additionalBytes, byte stackBehaviorPop, byte stackBehaviorPush, bool requiresStaticEnvToBeFalse = false)
```

Comment:
`public readonly struct` at least. Should we consider packing it up, like pop+push+isStatic? Maybe not needed. But having readonly can prohibit from copying.